x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -213,10 +265,39 @@ export function App({
           <div className="lnsApp__header">
             <TopNavMenu
               config={[
+                ...(!!originatingApp && lastKnownDoc?.id
+                  ? [
+                      {
+                        label: i18n.translate('xpack.lens.app.saveAndReturn', {
+                          defaultMessage: 'Save and return',
+                        }),
+                        emphasize: true,
```

Comment:
This changes the style of the button, right? It's now a lot more noticeable as a call to action.

Reply:
Exactly yes, the `emphasize` prop was added for the Create new button in the top nav of dashboard, so it's good that we're using it now in multiple places.